client/assets.coffee

```diff
@@ -146,18 +150,30 @@ emit = ($item, item) ->
       <input style="display: none;" type="file" name="uploads[]" multiple="multiple">
     """
 
+  assetsData = {}
+  $item.addClass 'assets-source'
+  $item.get(0).assetsData = -> assetsData
+
+  assets = item.text.match(/([\w\/-]*)/)[1]
+
+  if assets is 'PAGE'
+    if $item.parents('.page')[0].id.endsWith('template')
+      $item.append """
+        <div style="background-color:#eee;padding:15px; margin-block-start:1em; margin-block-end:1em;">
+          <p><i>Any pages created from this template will use a folder with their name for assets.</i></p>
```

Comment:
Better wording might say,
"will create assets in a folder with a name derived from the eventual page's title."

Reply:
Assets work when viewed through a revision. (I sometimes delete the Assets plugin for a cleaner look on the page then recall the revision from history when I want to change the upload.)

The new PAGE markup gets the derived folder name incorrect in this case by including the rev in the name.